From 3923f0df76d24b73d57f15eec61ab190ea048093 Mon Sep 17 00:00:00 2001
From: "Matthew \"strager\" Glazar" <strager.nds@gmail.com>
Date: Thu, 26 Oct 2023 18:08:30 -0400
Subject: [PATCH] fix(build): fix installing build tools for cross-compilation

'cmake --install . --component build-tools' copies no files [1]. This
was caused by commit 1f2e1a47 where the code calling install() became
dead code on accident. Call install() so that 'cmake --install' copies
the build artifacts as intended.

[1] https://github.com/quick-lint/quick-lint-js/issues/1099

Refs: 1f2e1a4701793cac24eaac44d7af81a8b820b1bc
---
 docs/CHANGELOG.md    | 7 +++++++
 tools/CMakeLists.txt | 1 -
 2 files changed, 7 insertions(+), 1 deletion(-)

 (docs/CHANGELOG.md changes omitted to reduce conflicts.)

diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
index 71ccbdf1b..b541afb52 100644
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -68,7 +68,6 @@ if (QUICK_LINT_JS_ENABLE_BUILD_TOOLS)
     COMMENT "Building all quick-lint-js build-time tools"
     DEPENDS ${QUICK_LINT_JS_BUILD_TOOL_TARGETS}
   )
-elseif (QUICK_LINT_JS_ENABLE_BUILD_TOOLS)
   install(
     TARGETS ${QUICK_LINT_JS_BUILD_TOOL_TARGETS}
     COMPONENT build-tools
